### DISCUSSION TITLE
Grove - Water Atomization | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#498] [Q&A] [answered]
# Grove - Water Atomization | Seeed Studio Wiki

Grove - Water Atomization is a fine Grove module for you to develop an atomizer or an atomizer module in your applications easily. With a few simple steps, you can prototype an atomizer. It has grove interface which make it easily applied to plenty of applications. A humidifier is a basic application it can be built with, you can develop more advanced and interesting objects with digital scent technology and any other situation in which atomization is required.

https://wiki.seeedstudio.com/Grove-Water_Atomization/

**Comment by TobiasReich** [comment]
I wanted to experiment with the Grove Water Atomizer however since I wanted to keep things compact, I tried to connect the device directly to the breadboard instead of using a “base shield” or such things.
So I understand the VCC pin needs to connect with the 5V out from my ESP32. The ground is obviously the ground.
However I have two additional connections at the atomizer driver board.
One is labeled NC and the other EN. In the sample code however only one pin is set to HIGH.
I understood from other tutorials that the EN connection ENables the atomizer. 

But when I set this pin to HIGH nothing happens!

Have I missed something? Do I have to do anything with the NC pin instead? There is no sound or LED that is lighting or anything. Is it broken or am I missing something?

**Reply by xzxcessarr** [reply, marked as answer]
Hi, thank you for feedback! After read your description, I think you can check by this list below:
1. check the ultrasonic transducer: please check whether the protrusion of sheet metal on ultrasonic transducer is dry and the opposite is wet, because the protrusion side is used for Atomizing and the opposite side is used for absorb water. if there are some water on protrusion side, you may see nothing happen whether everything else is well!
2. check the circuit: please enable the EN PIN high and connect the other two power supply pin well, then use the multimeter or other Voltage meter to connect the two pin interface on board and check whether the voltage is near 5V, if well, maybe your ultrasonic transducer have some problem; if the voltage is below this range even to the 0V, follow the next step:
3. check your esp32 board: make sure your esp32 board and pin work well by the same well above, the pin you connect to the EN pin is 3.3V and the pin using for power supply is 5V, if well, maybe your board or codes have some problem.
4. all check well ,you can just use the wet tissue on the sunken side of the ultrasonic transducer and try again, hope you can see the mist soon!

**Reply by TobiasReich** [reply]
Thanks for your reply. I took some time experimenting with it.
I checked all the voltages and it even seems to work now.
I guess the bigger issue is the hardware setup. When I "drip" a drop of water on it I can actually see some vapor. So things seem to work more or less. However it does not stay constant - even when tried with cotton or a sponge or such tings.
So not sure if I'm using it wrong or if there is something wrong. I always get only on-off results.

Gonna experiment with it a little further. Still thanks for your advice.

**Comment by Isaac-MML** [comment]
Hi,

I’m working on a project with the Grove Water Atomizer, and I have a couple of technical questions about the setup:

1. What is the atomization rate of the atomizer when the driver board is powered with a 5V supply?
2. Could you please provide the specific model and manufacturer of the ultrasonic transducer plate used in the atomizer?

Thank you for your help! I appreciate any details you can share about the hardware.
Best regards

**Reply by nfs0619** [reply]
Thanks for reaching out! The information we can provide about the Grove Water Atomizer is limited to what is available in the Resources section of the documentation. If you need more detailed information or have any further questions, please feel free to contact our technical support team at techsupport@seeed.io. They will be happy to assist you.

**Reply by Isaac-MML** [reply]
thank you very much
